kafka-ui-contract/src/main/resources/swagger/kafka-ui-api.yaml

```diff
@@ -172,6 +172,35 @@ paths:
                 items:
                   $ref: '#/components/schemas/BrokersLogdirs'
 
+  /api/clusters/{clusterName}/brokers/{id}/logDirs:
```

Comment:
Please do not use camel case in paths

Reply:
Url is copied from your task description :) But ok, I'll fix it.

Reply:
sorry, my bad)

Reply:
Fixed